readthedocs/doc_builder/director.py

```diff
@@ -65,7 +65,30 @@ def setup_vcs(self):
                 ),
             )
```

Comment:
These two conditionals will have moved to executing in the context manager. It doesn't seem important that either are executed outside the environment in the first place. But in case anyone has any more information about why this is, these seem to be the only logic order changes.

Reply:
This code was being already executed inside the context manager. The difference now is that instead of the context manager being executed inside the `BuildDirector` is executed in the Celery task, so we can call an extra method there (`sync_versions`), which needs to be executed inside the context manager (Docker environment)